libraries/aws_ssm_resource_compliance_summaries.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsSsmResourceComplianceSummaries < AwsResourceBase
+  name 'aws_ssm_resource_compliance_summaries'
+  desc 'Verifies settings for a SSM Resource Compliance Summary in bulk'
+  example '
+    describe aws_ssm_resource_compliance_summaries do
+      it { should exist }
+    end
+  '
+
+  attr_reader :table, :api_response
+
+  FilterTable.create
+             .register_column(:compliance_types,        field: :compliance_type)
+             .register_column(:resource_types,          field: :resource_type)
+             .register_column(:resource_ids,            field: :resource_id)
+             .register_column(:status,                  field: :status)
+             .register_column(:overall_severity,        field: :overall_severity)
+             .register_column(:execution_summaries,     field: :execution_summary)
+             .register_column(:compliant_summaries,     field: :compliant_summary)
+             .register_column(:non_compliant_summaries, field: :non_compliant_summary)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
```

Comment:
I believe adding server-side filtering would make this resource more efficient. 
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/SSM/Client.html#list_resource_compliance_summaries-instance_method
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/SSM/Types/ComplianceStringFilter.html